README.md

```diff
@@ -40,6 +37,15 @@ All absolute paths will now be converted and saved in snapshots like so:
 
 `/path/to/my-proj/lib` => `<PROJECT_ROOT>/lib`
 
+``/path/to/os-temp/nested/temp`` => ``<TEMP_DIR>/nested/temp``
+
+``/path/to/user-home/nested/home`` => ``<HOME_DIR>/nested/home``
+
+#### Caveats
+
+* All single backslashes (`\`) will be replaced by a forward slash (`/`)
+* Any string that looks like a Windows drive letter (`C:\`) will be replaced by a forward slash (`/`).
+
```

Comment:
I moved the note because I had added another, but later removed/solved it. Thought it was better left like this.